README.md

```diff
@@ -47,7 +47,7 @@ details.
 - The OpenDDS Developer's Guide is freely downloadable at:
 http://download.objectcomputing.com/OpenDDS/
 
-- The TAO Developer's Guide book set may also be purchased from
+- The TAO Developer's Guide book can be requested for free from
```

Comment:
Should it move to the ACE/TAO section of the document?

Reply:
I would just keep it here

Reply:
I would agree with moving it. We could maybe also just remove it entirely if we have other links were someone could find it through if they wanted to know more about ACE/TAO.

Reply:
Moved